src/classes/storage.ts

```diff
@@ -1,24 +1,55 @@
-import { StopGroup, StopCluster, StopGroupGpuid, StopClusterGpuid } from '../types';
+import {
+  StopGroup,
+  StopCluster,
+  GroundPlacesList,
+  GroundPlaceGenerated,
+  StopGroupGpuid,
+  StopClusterGpuid,
+} from '../types';
 
 /**
  * @description Manipulate GroundPlaces.
  */
 export class Storage {
```

Comment:
The storage should have a read method. That read the json file.
Then you'll be able to mock efficiently

Reply:
Done